src/explore-education-statistics-admin/src/components/Page.tsx

```diff
@@ -45,7 +45,9 @@ const Page = ({
           'dfe-width-container--wide': wide,
         })}
       >
-        {pageBanner ?? <PageBanner />}
+        {pageBanner ?? (
+          <PhaseBanner url="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-VQ56HAfKLpBrG0LxbfxbVdUQjVJQVdMOFlSMURGQ1kyMzRNWlpKN1NMVy4u" />
```

Comment:
Just a minor thought, but I wonder if it's worth abstracting this URL to somewhere? 

In previous projects I've seen things like this all built up a `Constants.ts` file, or pulled out of config. Only a thought/suggestion to help readability/maintainability, definitely not a blocking comment :-)

Reply:
Yeah that's worth considering, I'll not do it for this hotfix though.